laterpay/application/Controller/Install.php

```diff
@@ -519,4 +520,57 @@ public function update_default_pricing_behaviour() {
             delete_option( 'laterpay_only_time_pass_purchases_allowed' );
         }
     }
+
+    /**
+     * Init ga options.
+     * Add options needed for sending Google Analytics Events.
+     *
+     * @since 2.2.0
+     *
+     * @return void
+     */
+    public function init_ga_options() {
+        $current_version = get_option( 'laterpay_plugin_version' );
+        // @todo: change version compared to `2.1.0`
+        // @todo: Only Add / Update option only if current one doesn't exist.
+        if ( version_compare( $current_version, '2.0.0', '<' ) ) {
+            return;
+        }
+
+        $lp_personal_data = [];
+        $lp_tracking_data = [];
+
+        $lp_personal_data['laterpay_ga_personal_enabled_status'] = 0;
+        $lp_personal_data['laterpay_ga_personal_ua_id']          = '';
+
+        if ( is_multisite() ) {
+            $monster_insights_network_profile = get_site_option( 'monsterinsights_network_profile', array() );
```

Comment:
Should add plugin URL somewhere in a comment and what we are doing if that plugin is activated.